auto_submit/lib/service/pull_request_validation_service.dart

```diff
@@ -104,14 +177,92 @@ class PullRequestValidationService extends ValidationService {
     for (MapEntry<String, ValidationResult> result in validationsMap.entries) {
```

Comment:
for (MapEntry(:key, :result) in validationsMap.entries) {
    // now you could use key and result thanks to patterns
}